### PR TITLE
docs: update jstz installation doc

### PR DESCRIPTION
**Pull request by Frank-en-stein**
Updated jstz docs/Installation.md with the latest installation steps.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
...
-:::
+### llvm 📜
 
-#### Rust 🦀
+Install llvm. For instance, on MacOS, run:
 
-> `jstz` requires a specific release of Rust. The version of Rust required is specified in the `rust-toolchain` file.
+```sh
+brew install llvm
+```
```
Comment (johnyob) [inline docs/installation.md:33]: I'd opt for avoiding OS specific installations (either we provide both Linux & MacOS instructions) or we link to a page about installing them

File: docs/installation.md [hunk 2]
```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
...
 ```sh
 git clone https://github.com/trilitech/jstz.git
+git clone https://gitlab.com/tezos/tezos.git
 ```
 
 ### Prerequisites 📋
 
-::: tip  
-Both `jstz` and Octez are packaged with Nix, a package manager and system configuration tool that makes building from sources easy! See the [Nix docs](https://nixos.org/download.html) for instructions for your system. Additionally, ensure [Nix flakes are enabled](https://nixos.wiki/wiki/Flakes#Enable_flakes).
-:::
```
Comment (johnyob) [inline docs/installation.md:26]: Nix isn't necessarily a pre-requisite, but an optional one that makes things slightly easier (especially for those on MacOS). I'd prefer to keep it in the 'tips'

File: docs/installation.md [hunk 3]
```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
...
 
-### Cloning the Repository
+### Cloning the Repositories
+
+Get the source code for both `jstz` and Octez.
 
 ```sh
 git clone https://github.com/trilitech/jstz.git
+git clone https://gitlab.com/tezos/tezos.git
 ```
```
Comment (johnyob) [inline docs/installation.md:23]: You won't always have to build Octez from sources. 

Historically we did was due to floating point support and we only currently require sources since the `master` version of Octez is buggy

File: docs/installation.md [hunk 4]
```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
...
 
-After Nix successfully builds Octez (it takes a long time 🕣), the Octez binaries will be accessable from `result`.
-:::
+After Nix successfully builds Octez (which takes a long time 🕣), the Octez binaries will be available in the `result/bin/` directory. Copy the following binaries to the `jstz` root direactory:
+
+- octez-client
+- octez-node
+- octez-smart-rollup-node
+- octez-smart-rollup-wasm-debugger
```
Comment (johnyob) [inline docs/installation.md:79]: `octez-smart-rollup-wasm-debugger` should be dropped. Not required for the sandbox and is no-longer a useful tool for us

File: docs/installation.md [hunk 5]
```diff
@@ -88,8 +102,7 @@ You can locate the resulting build artifact at `/target/wasm32-unknown-unknown/r
 You can now start the sandbox with:
 
 ```sh
-make build-installer
-cargo run -- sandbox start
+cargo run --bin jstz -- sandbox start
```
Comment (johnyob) [inline docs/installation.md:105]: Some of the build commands have changed since this was written 😅

I think it should be:
```sh
make build-cli-kernel
cargo run --bin jstz -- sandbox start
```

File: docs/installation.md (hunk 3 above)
Comment (Frank-en-stein) [reply docs/installation.md:23]: That makes sense but should we keep it like this until we're out of the woods? Leaving building from source out of the docs as tribal knowledge would be harder for anyone trying to onboard to jstz.

Comment (johnyob) [reply docs/installation.md:23]: I'm not suggesting we leave it out, but cloning from sources for `tezos/tezos` is a step in installing Octez in the Octez documentation. 

We should avoid duplication between these two docs (only offering suggestions e.g. nix that might aid the user)

**Review by johnyob — COMMENTED** [review]
See review commit. 

TL;DR:
- Keeping Nix optional 
- Adding a code-group for LLVM installation instructions
- Updating commit shas / branch names to tezos/tezos

File: docs/installation.md [hunk 6]
```diff
@@ -26,13 +24,39 @@ git clone https://github.com/trilitech/jstz.git
 Both `jstz` and Octez are packaged with Nix, a package manager and system configuration tool that makes building from sources easy! See the [Nix docs](https://nixos.org/download.html) for instructions for your system. Additionally, ensure [Nix flakes are enabled](https://nixos.wiki/wiki/Flakes#Enable_flakes).
 :::
 
-### Clang and llvm 📜
+#### LLVM 🛠️
+
```
Comment (Frank-en-stein) [inline docs/installation.md:28]: ìs it better to have a copy of the installation guide? I was looking for modularised section but I suppose can't do it in md

**Review by johnyob — APPROVED** [review]
